### ISSUE TITLE
csso removes space after calc() closing parenthesis

### ISSUE THREAD

**Issue by seleckis** [css/csso#308]
e.g.:
```css
border: calc(1vh + 1vw) solid transparent;
```
produces:
```css
border:calc(1vh + 1vw)solid transparent;
```
Most browsers read it correctly except IE11:
![screen shot 2016-07-08 at 14 59 11](https://cloud.githubusercontent.com/assets/988504/16686695/9536e7f8-451c-11e6-936d-25cc039c4c4c.png)
So IE11 strikes through all the properties of the selector where the bug has been found.

**Comment by lahmatiy** [comment]
What version do you use?
Issue was fixed in CSSO 1.8.1 #289

**Comment by seleckis** [comment]
Yes, the problem was in old version. Just updated gulp-csso, now everything is ok. Thank, you.
